pkg/ui/src/components/link/link.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+import "./link.styl";
+
+interface LinkProps {
+  href: string;
+  target?: "_blank" | "_parent" | "_self";
+}
+export function Link(props: React.PropsWithChildren<LinkProps>) {
```

Comment:
Can we call this something other than Link to disambiguate from Router Link?

Reply:
Done, renamed to `Anchor`.